CHANGELOG.rst

```diff
@@ -9,6 +9,11 @@ This document describes changes between each past release.
 **Breaking changes**
 
 - Upgrade to kinto-http 8.0.0 with Python 3.5+ support.
+- The ``--full`` option of the ``dump`` command now outputs records (#16)
```

Comment:
Well, if you rely on the ``dump`` command and don't expect it to include records, it would be a breaking change